test/functional/test-3p.js

```diff
@@ -147,14 +147,12 @@ describe('3p', () => {
       }, /* mandatory */[], ['foo', 'bar']);
       clock.tick(1);
 
-      validateData({
-        type: 'TEST',
-        foo: true,
-        'not-whitelisted': true,
-      }, [], ['foo']);
-
       allowConsoleError(() => { expect(() => {
-        clock.tick(1);
+        validateData({
```

Comment:
i guess you can remove all the clock ticks above in this test.

Reply:
Done.